.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+# Description
+Briefly describe the problem or reason for the changes implemented in the PR
+
+# Trac ticket
+Link to the accepted Trac ticket for this PR. All PRs except docs or tests changes need an accepted ticket. New features need community consensus.
+
+# Tests
+Briefly describe any tests you have added or updated to test the changes introduced by the PR
+
+# Checklist
+- [ ] I have updated the Trac ticket’s flags
```

Comment:
As Trac isn't super intuitive, I think it would be helpful to tell users what to do exactly (ie set "has patch" on the ticket).

Reply:
```suggestion
- [ ] I have updated the "has patch" Trac ticket
```